recipes/mljar-scikit-plot/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mljar-scikit-plot" %}
+{% set version = "0.3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e9a6a6c6fdfa277a0ced22872350336556d3c8022f3618ad0c7c0ed236069746
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
```

Comment:
The package classifiers say 3.8 but there is not a python_requires info.

Reply:
I updated the package https://github.com/mljar/mljar-scikit-plot/blob/master/setup.py

Reply:
NP. Consider using the `python_requires` metadata. Classifiers don't mean anything to a package.

Reply:
Thanks! :1st_place_medal:  Fixed.